src/main/domain/ping/PingReturn.js

```diff
@@ -32,21 +30,21 @@ class PingReturn {
    * @returns {Boolean}
    */
   get cmpLoaded() {
-    return this._cmpStatus.loaded
+    return this._status.loaded
   }
 
   /**
    * see Ping Status Codes in following table
    */
   get cmpStatus() {
-    return this._cmpStatus.code
+    return this._status.cmpStatus
   }
 
   /**
    * see Ping Status Codes in following table
    */
   get displayStatus() {
-    return this._displayStatus.code
+    return this._status.cmpStatus
```

Comment:
did you mean this._status.displayStatus ?

Reply:
Fixed!